README.md

```diff
@@ -1,93 +1,284 @@
 
-For federated and incubating projects, the Project Team decides among themselves on how to add and/or remove members from the Project Team. Further, Project Teams for incubating projects are exempted from the requirement to have at least one Steering Council member. For a community maintained project, a petition for commit rights is made to the current members of the Project Team for that project. If this petition passes according to the voting rules for "Nominate new member of a Project Team" below, the petitioner is granted write access and added to the team.
+In the other example, Microsoft is a large company and these three largely unrelated parts of it are unlikely to have common goals based mainly on their common ultimate funder. However, a company that wanted to control the conda Organization could do so by enlisting people from across their organization, claim separate funding, and then capture the council. **In this case, the default guideline is to treat all parts of a company as having one funder.**
 
-**Steering Council** members are excluded from these votes and retain commit rights to all community projects as stated above. They can add or remove themselves from Project Teams as they see fit in order to indicate which projects they are involved with, have a say in voting for write access petitions, etc. Steering Council members may not add themselves to a Project Team in order to vote on a petition for commit access if they were not already on that team when the petition was made.
+#### Tracking Funding
 
+Nominated members will provide a list of organizations that fund above 25% of their time. Nominees will need to provide a complete “chain of funding” for each funding source. For example: “GitHub, a part of Microsoft”, or “CS Dept at Western Michigan University, *Computational Reproducibility Made Easy grant*, NHGRI, NIH, US Federal Government.”
+
+The Steering Council determines the appropriate funding level(s) to consider when looking for common funding with existing and proposed members. The funding source of any Steering Council member can be put to a vote under the policy for [All Other Items](#all-other-items) below.
+
+Council members need to keep their funding documentation up to date, and notify the council whenever it changes. Should a change in funding state create a shared funding situation, the council members should decide who among themselves should move to emeritus status. If no agreement can be reached, the Steering Council will hold a simple vote to determine which member is moved to emeritus status under the [Resolve Overlapping Funding](#resolve-overlapping-funding) voting provision below.
+
+In cases where people have absolutely no funding related to conda, we still document this funding state, but we do not require people to list their (irrelevant) funding. The "no funding" state can be held by an unlimited number of Steering Council members.
```

Comment:
How is this "no funding" determined? If someone works on Conda to patch bugs found in the pursuit of a particular grant were they funded to work on Conda?

I am making this comment solely in my personal capacity and am not conveying any rights to any intellectual property of any third parties.

Reply:
@CJ-Wright It's clear to me (in my brain!) that the answer to this specific question is "No," the person is not funded to work on conda.  However, my brain is not part of this document.

What about:

`In cases where people have absolutely no funding related to conda, we still document this funding state, but we do not require people to list their (irrelevant) funding. The "no funding" state can be held by an unlimited number of Steering Council members.`

`What about cases where someone works on conda for work they are doing, but improving/directing the conda ecosystem is not a specific component of any of their funding?  Generally, such cases should be considered as "no funding."`

`(And then we will thank these people! Volunteers are a big part of the conda community and many components of the ecosystem would not exist without their contributions. The conda community absolutely wants these contributors involved in the community.`

Ok, that gets a little preachy towards the end. We could not include that last paragraph.

Reply:
I think that `improving/directing the conda ecosystem is not a specific component of any of their funding` can reasonably be argued to cover most folks.

Reply:
@CJ-Wright & @awwad: More descriptive text has been added.